src/LanguageServer/ProtocolUnitTests/Workspaces/LspWorkspaceManagerTests.cs

```diff
@@ -229,51 +229,6 @@ public async Task TestLspFindsNewDocumentAsync(bool mutatingLspWorkspace)
         Assert.Equal(testLspServer.TestWorkspace.CurrentSolution, lspDocument.Project.Solution);
     }
 
-    [Theory, CombinatorialData]
-    public async Task TestLspTransfersDocumentToNewWorkspaceAsync(bool mutatingLspWorkspace)
```

Comment:
This was moved to `AbstractLspMiscellaneousFilesWorkspaceTests.cs` and reworked right?